apollo-router/src/query_planner/testdata/starstuff.graphql

```diff
@@ -0,0 +1,98 @@
+schema
```

Comment:
I think we already have a number of copies of this schema. Is this one different? Or could an existing file be reused instead?

Reply:
I can point to an existing copy. I'll update the code.